index.js

```diff
@@ -1,136 +0,0 @@
-               process.exit()
-          }
-     } else if (cheatCode === 2) { // check if human lied about number beeing higher
-          if (guessLowerThan - 1 === currentGuess) {
-               console.log("\nStupid Human\nYou already said it was lower than " + guessLowerThan + ".\nSo it cannot also be higher than " + currentGuess + ".")
-               process.exit()
-          }
-     } else if (cheatCode === 3) {  // check if human lied about number beeing lower
-          if (guessHigherThan + 1 === currentGuess) {
-               console.log("\nStupid Human\nYou already said it was higher than " + guessHigherThan + ".\nSo it cannot also be lower than " + currentGuess + ".")
-               process.exit()
-          }
-     }
-}
-
-start()
-
-// Start game and choose role of computer and human - who picks number and who guesses - launch game version
-async function start() {
-     console.log("\nLet's play a game where one of us picks a number inside of a range\nand the other tries to guess it.\n")
-     gameVersion = await ask("Do you want to be the number picker or number guesser?\nEnter P for picker\nEnter G for guesser\n>")
-     if (gameVersion === 'P') {
-          computerGuesses()
-     } else if (gameVersion === 'G') {
-          humanGuesses()
-     } else {
-          console.log("Pick capital P or capital G only.\nPlease start over.")
-          process.exit()
-     }
-}
-
-// Run Human Guesses game
-async function humanGuesses() {
-     console.log("\nNow I (Mighty computer) will pick a number between 1 and 100\nand You (Puny human) try to guess it.\n")
-     hiNum = 100
-     secretNumber = randomInteger(loNum, hiNum)
-     numOfGuesses = 0
-
-     // start loop and run until human guess is correct
-     while (currentGuess !== secretNumber) {
-          currentGuess = Number(await ask("What is your guess ?\n>"))
-          numOfGuesses += 1
-          if (currentGuess > secretNumber) {
-               console.log("Guess lower")
-          } else if (currentGuess < secretNumber) {
-               console.log("Guess higher")
-          }
-     }
-
-     // win condition satisfied - computer acknowledges same and exits
-     console.log(secretNumber + " is my secret number.\n")
-     if (numOfGuesses <= 7) {
-          console.log("You must have cheated to guess it in only " + numOfGuesses + " tries.")
-     } else {
-          console.log("HA!\nYou guessed it but it took you " + numOfGuesses + " tries.")
-     }
-
-     process.exit()
-}
-
-// Run computer guesses game and run until human says number guessed is correct
-async function computerGuesses() {
-     numOfGuesses = 1
-     console.log("\nNow You (Puny human) will pick a number and\nI (Mighty computer) will certainly guess it.\n")
-     // Allow Human to set range
-     hiNum = +(await ask("\nI will even allow you to set the range.\nLet's say bettween 1 and...?\nEnter a number greater than 1 \n>"))
```

Comment:
Adding an `if` statement here could be used to prevent the user from putting something other then a number in. As of right now, if you put a word in it will still continue with the game